packages/react/jest.config.ts

```diff
@@ -2,13 +2,13 @@ import type { Config } from 'jest';
 
 const config: Config = {
   testEnvironment: 'jsdom',
-  setupFilesAfterEnv: ['<rootDir>__tests__/setupTests.ts'],
+  setupFilesAfterEnv: ['<rootDir>/__tests__/setupTests.ts'],
   testMatch: [
     '**/__tests__/src/**/*.js',
     '**/__tests__/demo/**/*.js',
     '**/src/components/**/*.test.tsx'
   ],
-  collectCoverageFrom: ['src/**/*.{ts.tsx}'],
+  collectCoverageFrom: ['<rootDir>/src/components/**/*.{tsx,js}'],
```

Comment:
Whoops, typo on my part, good catch! 🤦 

This should probably be:

```suggestion
  collectCoverageFrom: ['<rootDir>/src/components/**/*.{tsx,ts}'],
```

Since there are `.ts` files that are not just index files. Are there actually any js files located under components? I couldn't find any.

Reply:
@scurker I double-checked and didn't find any. Updated!